brunokiafuka/react-native-chrono
# chore: config react-native-chrono as library

### Description 
This PR fixes #10 by
- Adding new lib set up using: https://callstack.github.io/react-native-builder-bob/build
- Used monorepo setup
- Converted lib to example
- Added docs
- Fixed lib components structure

### How to test
- Check out `convert-to-lib`
- Install dependencies under the root dir and `./example` - this will automatically link the example with lib
- Run `yarn example start` to run the example app, in case you need to clear your metro cache run `yarn example start -c` instead

### Example app using the set up

https://github.com/brunokiafuka/react-native-chrono/assets/12303468/2f51e5d6-ecc5-40b6-b5b6-8a5936cf4eb7
